go.mod

```diff
@@ -220,6 +220,8 @@ replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-d
 // If you change this please also update docs/INSTALL.md and GO_VERSION in
 // Makefile (then run `make lint` to see where else it needs to be updated as
 // well).
-go 1.23.12
+go 1.24.6
 
 retract v0.0.2
+
+replace github.com/btcsuite/btcwallet => github.com/ziggie1984/btcwallet v0.16.10-0.20250902074924-f9f0dabe6fbd
```

Comment:
I lack experience with this in `lnd`, but are we ok with merging replaces to local forks in `lnd` `master`?